<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Time">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>651</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_texttaikhoan">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;lt;font color = red&amp;gt;&amp;lt;font&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&lt;font color = red&gt;Tên đăng nhập không được bỏ trống!&lt;font&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_textmatkhau">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>581</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;font color = red&gt;Mật khẩu không được bỏ trống!&lt;font&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Login">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>254</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Đăng nhập </string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image_no_color/icon_no_color/angle-double-right.svg</normaloff>:/image_no_color/icon_no_color/angle-double-right.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>38</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Return">
     <property name="geometry">
      <rect>
       <x>468</x>
       <y>254</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Về trang chủ</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image_no_color/icon_no_color/back-left.svg</normaloff>:/image_no_color/icon_no_color/back-left.svg</iconset>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>581</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tên tài khoản:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Taikhoan"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>120</y>
       <width>581</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mật khẩu:       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Matkhau">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:x-large;&quot;&gt;ĐĂNG NHẬP VÀO HỆ THỐNG &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
